app/src/main/java/br/org/cesar/discordtime/stickysessions/presentation/lobby/LobbyPresenter.java

```diff
@@ -40,6 +52,15 @@ public void onCreateGainPleasure() {
         mCreateSession.execute(new CreateSessionObserver(), SessionType.GAIN_PLEASURE);
     }
 
+    private void goNext(String event){
+        try {
+            mRouter.goNext(LobbyActivity.NAME, event);
```

Comment:
I think View interface could have a `getName()` method. In that way, we can still test presenters using JUnit instead of Robolectric.